src/Graphics/OpenGLContext/GLFunctions.cpp

```diff
@@ -45,7 +45,7 @@ static void* AppleGLGetProcAddress (const char *name)
 	return (image ? dlsym(image, name) : NULL);
 }
 #define glGetProcAddress AppleGLGetProcAddress
-#define GL_GET_PROC_ADR(proc_type, proc_name) ptr##proc_name = (proc_type) glGetProcAddress(#proc_name)
+#define GL_GET_PROC_ADR(proc_type, proc_name) ptr##proc_name = (proc_type) glGetProcAddress("gl"#proc_name)
```

Comment:
Current code works on MacOS. Why you changed it? Have you tested that change?

Reply:
I have not tested the change, I don't have the right devices to be able to test it. Apparently this fixed Linux though, and the same mistake _appears_ to be in MacOS as well.

I changed it because I see what looks like a mistake. Without the "gl" in front, it looks like it will attempt to get the function name incorrectly. Maybe MacOS is ok with that?

Reply:
Hmm, yes, it looks wrong without "gl".
However, I have tested my own build of GLideN64 on a Macbook several month ago and it worked.
Ok, lets fix only Linux for now. I'll try to get a Macbook again and test with and without that fix.